tests/fixtures/optimizer/optimizer.sql

```diff
@@ -39,13 +39,13 @@ WITH "cte" AS (
     SELECT
       "x"."a" AS "a"
     FROM "x" AS "x"
-  ) AS "_q_0"
```

Comment:
why did this get changed? is this correct?

Reply:
These statements were getting treated as derived tables, so they get qualified.

But I think they're actually subqueries, so I think this is correct.